README.md

```diff
@@ -214,6 +214,7 @@ If you’re using a framework or package manager that doesn’t support npm, you
 
    You'll notice in our example above that we also outline a `stylesheets`, `images` and `javascript` folder in your `assets` folder. These folders are to help organize any assets that are unique to your project and separate from the design system assets.
 
+   **Note:** The file date defaults to October 26, 1985, when you download and install this package. This is intentional. NPM sets a default date to ensure reproducible builds for npm packages.
```

Comment:
When using npm at the beginning of a sentence, capitalize ("Npm"). There are other instances on this page that should be addressed, too. Shall we handle them here or on a new PR?

Reply:
Yeah I think we should handle the cleanup of `npm` in separate PR's so this work is directly scoped to this note. 👍

Reply:
Resolved in https://github.com/uswds/uswds/pull/5871/commits/0c85083f1a9b2c4be94a886afa102e310c74720d